cmd/kubeapps-apis/proto/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2.proto

```diff
@@ -7,10 +7,16 @@ import "kubeappsapis/core/packages/v1alpha1/packages.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 service FluxV2PackagesService {
-  // GetAvailablePackages returns the available packages managed by the 'fluxv2' plugin
-  rpc GetAvailablePackages(kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesResponse) {
+  // GetAvailablePackageSummaries returns the available packages managed by the 'fluxv2' plugin
+  rpc GetAvailablePackageSummaries(kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackageSummariesResponse) {
     option (google.api.http) = {
-      get: "/plugins/fluxv2/packages/v1alpha1/packages"
+      get: "/plugins/fluxv2/packages/v1alpha1/availablepackagesummaries"
+    };
+  };
+  // GetAvailablePackageDetail returns the package metadata managed by the 'fluxv2' plugin
+  rpc GetAvailablePackageDetail(kubeappsapis.core.packages.v1alpha1.GetAvailablePackageDetailRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackageDetailResponse) {
+    option (google.api.http) = {
+      get: "/plugins/fluxv2/packages/v1alpha1/availablepackagedetails"
```

Comment:
```suggestion
      get: "/plugins/fluxv2/packages/v1alpha1/availablepackagedetail"
```
maybe?

Reply:
Well, neither is really correct indeed, we should move to something more rest-ish, like `/plugins/fluxv2/packages/v1alpha1/availablepackagedetails/{packageID}`
or even `/plugins/fluxv2/packages/v1alpha1/availablepackages/{packageID}/detail`, I don't think?

But, given that the details refer to a specific package, the package id should be part of the URL somehow.

However, we can discuss it in the design doc and change all of them in batch once we reach an agreement.

Reply:
> Well, neither is really correct indeed, we should move to something more rest-ish, like `/plugins/fluxv2/packages/v1alpha1/availablepackagedetails/{packageID}`
> or even `/plugins/fluxv2/packages/v1alpha1/availablepackages/{packageID}/detail`, I don't think?

Yes, I'd love to (in another PR) use the latter, but it's going to be tricky as that isn't a unique URL on its own without also including the context in the URL, like:

```
/plugins/fluxv2/packages/v1alpha1/c/default/ns/my-namespace/availablepackages/{packageID}/detail
```

but both the cluster and the namespace can be empty. Actually, we can ensure the cluster is always present easily enough (after the recent work you did to provide the cluster config), so it's just the namespace. I don't know if the protobuf annotations for specifying the URL can cope to additionally serve
;

```
/plugins/fluxv2/packages/v1alpha1/c/default/availablepackages/{packageID}/detail
```
so we can support both namespaced and cluster-scoped packages. Need to check. Otherwise it might need some sucky illegal namespace name (`.../ns/_cluster-scoped/availablepackages/...`). 

> 
> But, given that the details refer to a specific package, the package id should be part of the URL somehow.
> 
> However, we can discuss it in the design doc and change all of them in batch once we reach an agreement.

Yes - sounds good. So back to the suggestion here... I'd only balked at the plural because it read as "available package (details)" (as in details of an available package), but I see now you meant the rest-ish "(available package detail)s" (as in many of the "available package detail"). Let's stick with yours for now as it's at least more correct than the suggestion :)

Reply:
I'd go with something like `/plugins/fluxv2/packages/v1alpha1/c/{namespace}/availablepackages/{packageID}/detail`  (adding the cluster as a path param, but not the namespace).

The gRPC docs says: 
> Any fields in the request message which are not bound by the path template automatically become HTTP query parameters if there is no HTTP request body.

So I don't foresee any problems in not using the namespace as it can be empty.

> Let's stick with yours for now as it's at least more correct than the suggestion :)

Great, thank you!   Naming is always really tricky; besides, we are `AvailablePackageDetails` is not an easy word. When using RPC naming is just a matter of thinking of operations/actions, but back to the rest world, we should move to resources and use nouns instead of verbs... but sometime these nouns are tricky haha.